.github/workflows/security_owasp.yml

```diff
@@ -2,19 +2,11 @@ name: Security OWASP dependency check
 on:
   workflow_dispatch:
   schedule:
-    - cron: "20 5 * * MON-FRI" # Every weekday at 05:20 UTC
+    - cron: "0 5 * * MON-FRI" # Every weekday at 05:00 UTC
 jobs:
-  security-kotlin-owasp-check-search:
-    name: Search Kotlin security OWASP dependency check
-    uses: ministryofjustice/hmpps-github-actions/.github/workflows/security_owasp.yml@v0.5 # WORKFLOW_VERSION
+  security-kotlin-owasp-check:
```

Comment:
I'm assuming this means it's clever enough to work out there are two sub-projects?